tests/RuleSet/RuleSetTest.php

```diff
@@ -376,7 +376,7 @@ public function testDuplicateRuleConfigurationInSetDefinitions(): void
 
             static::fail($message);
         } else {
-            $this->addToAssertionCount(1);
+            $this->expectNotToPerformAssertions();
```

Comment:
this one was correct, the assert is above it (`if (\count($duplicates) > 0) {`) , it passed so increase of the count seems valid to me